src/Dialogs/SettingsDialog.vala

```diff
@@ -186,6 +183,96 @@ public class Akira.Dialogs.SettingsDialog : Gtk.Dialog {
         return grid;
     }
 
+    private Gtk.Widget get_about_box () {
+        var grid = new Gtk.Grid ();
+        grid.row_spacing = 6;
+        grid.column_spacing = 12;
+        grid.halign = Gtk.Align.CENTER;
+
+        var app_icon = new Gtk.Image ();
+        app_icon.gicon = new ThemedIcon ("com.github.akiraux.akira");
```

Comment:
We can use `Config.APP_ID` here

Reply:
Definitely! Icon is not render on flatpak

![Captura de pantalla de 2020-04-17 10-11-18](https://user-images.githubusercontent.com/220968/79547305-fd832680-8093-11ea-8cc2-cbeca371ce9d.png)